rollup.config.js

```diff
@@ -8,6 +9,11 @@ const plugins = [
   babel({
     exclude: 'node_modules/**',
   }),
+  copy({
+    targets: [
+      { src: 'src/index.d.ts', dest: 'lib/' },
+    ],
+  }),
```

Comment:
We can use the copy plugin to, well, copy over the type definitions file into the `lib/` directory.